client/src/App.jsx

```diff
@@ -131,7 +118,7 @@ function CentralContentContainer({ user, socket }) {
           />
         </CompatRoute>
         <CompatRoute path="/datasets">
-          <Redirect to="/search?type=dataset" />
+          <Redirect to={`${ABSOLUTE_ROUTES.v1.search}?type=dataset`} />
```

Comment:
[non-blocking] This works too:

```suggestion
          <Redirect
            to={{ pathname: ABSOLUTE_ROUTES.v1.search, search: "type=dataset" }}
          />
```